### DISCUSSION TITLE
scene group for tile map support for changeing export vars

### DISCUSSION THREAD

**Discussion by ExtraBytez** [godotengine/godot-proposals#9738] [Editor]
### Describe the project you are working on

zelda 1 clone

### Describe the problem or limitation you are having in your project

cant change enemy strength

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

add a way to change export vars this way

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

like how would usually change exported vars in the inspector but for a tilemap sceen group object

### If this enhancement will not be used often, can it be worked around with a few lines of script?

no

### Is there a reason why this should be core and not an add-on in the asset library?

idk

**Comment by ExtraBytez** [comment]
sorry for sloppyness m made this on steamdeck

**Comment by ExtraBytez** [comment]
will this be implemented?

**Reply by AThousandShips** [reply]
Thed details you've provided are very limited, and it's only been three days, please have patience

**Reply by ExtraBytez** [reply]
ok ill add some screen shots  sorry for not sounding patient thats my fault 
![Screenshot 2024-05-16 120741](https://github.com/godotengine/godot-proposals/assets/99703235/e9d98b6f-8862-4ea2-8683-cd02c6b7a0a4)
![Screenshot 2024-05-16 120828](https://github.com/godotengine/godot-proposals/assets/99703235/065fb8fb-c320-4dd0-bf42-b9a93bd11520)
![Screenshot 2024-05-16 120839](https://github.com/godotengine/godot-proposals/assets/99703235/e0ac5f64-5cb4-4e47-a5e7-7d772b616d8a)

**Comment by dalexeev** [comment]
Note that you cannot set properties for each individual **cell**, only for the tile **type**. This applies to both Custom Data Layers and Scene Tiles, since the cell (for each layer) is mapped to the tile id (int, Vector2i, int). The cell has no metadata as far as I know.

The feature has already been proposed, see:

* #6672
* #7249

Let's close this discussion for consolidation. Please use the search before opening a proposal in the future. Thanks for the contribution nonetheless!

**Reply by ExtraBytez** [reply]
sorry 😅
